docs/index.md

```diff
@@ -135,14 +135,14 @@ Next, define a set of services using `docker-compose.yml`:
     redis:
       image: redis
 
-This defines two services:
+This template defines two services:
```

Comment:
Thanks @moxiegirl, looks good. (I am still a collaborator, so it doesn't look like I can "edit" the comment.)

However, the changes are pretty straightforward, so I will manually incorporate these updates and update my PR shortly.